app/frontend/helpers/sanitizeMnemonic.js

```diff
@@ -0,0 +1,3 @@
+const sanitizeMnemonic = (mnemonic) => mnemonic.replace(/,/g, ' ').replace(/\s+/g, ' ')
```

Comment:
I would put there a comment explaining what those regexes do since in general regexes are hard to read